agent/agent.go

```diff
@@ -3462,6 +3462,13 @@ func (a *Agent) registerCache() {
 		// Prepared queries don't support blocking
 		Refresh: false,
 	})
+
+	a.cache.RegisterType(cachetype.NodeServicesName, &cachetype.NodeServices{
+		RPC: a,
+	}, &cache.RegisterOptions{
+		// Prepared queries don't support blocking
+		Refresh: false,
```

Comment:
This should be able to support refreshing.

```suggestion
		Refresh: true,
```

Additionally the comment above should be updated.